### DISCUSSION TITLE
Email push subscription

### DISCUSSION THREAD

**Discussion by kilir** [mtex-toolbox/mtex#521] [Ask Anything]
just in case somebody masters the github settings: Is there any way of having an email subscription to this discussion? So far, I could not find anything except getting a notification for a thread I'm participating in.

Cheers,
Rüdiger

**Comment by ralfHielscher** [comment]
Hi Rüdiger,

currently the only possibility is to watch the entire repository and get notified not only about all discussions but also about all issues. There is a pending feature request to the Github team to make notifications specific to the discussion type.

I hope they get this implemented soon.

Ralf.
